packages/tux/src/components/fields/DatePicker.tsx

```diff
@@ -95,12 +96,14 @@ class DatePicker extends Component<any, State> {
           onFocus={this.handleInputFocus}
           onBlur={this.handleInputBlur}>
           <input
+            id="dateInput"
```

Comment:
We need to generate the ID to support having many components in the same page. This is a global variable.